packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurface.ts

```diff
@@ -17,6 +17,7 @@ export const usePopoverSurface_unstable = (
   props: PopoverSurfaceProps,
   ref: React.Ref<HTMLDivElement>,
 ): PopoverSurfaceState => {
+  const id = 'popover-' + useId();
```

Comment:
useId lets you add a prefix, it's better to let useId handle that.
```suggestion
  const id = useId('popoverSurface-');
```